app/views/unattended/provisioning_templates/PXELinux/windows_default_pxelinux.erb

```diff
@@ -4,8 +4,10 @@ name: Windows default PXELinux
 model: ProvisioningTemplate
 oses:
 - Windows
-%>
+description: |
+  TODO
```

Comment:
@UXabre any chance you could put together a description for this template?

Reply:
I think we can just drop a generic one, PXELinux to boot Windows.

Maybe adding a note that Windows boot files are not preconfigured by default by our installer.